src/Discord.Net.Interactions/Results/PreconditionGroupResult.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Discord.Interactions
+{
+    /// <summary>
+    ///     Represents a result type for grouped command preconditions.
+    /// </summary>
+    public class PreconditionGroupResult : PreconditionResult
+    {
+        /// <summary>
+        ///     Results of the preconditions of this group
```

Comment:
Should be prefixed with `Gets`

Add missing `.` to end of summaries (applies to whole file)